bigquery/google/cloud/bigquery/job.py

```diff
@@ -967,6 +968,20 @@ def destination_encryption_configuration(self, value):
             api_repr = value.to_api_repr()
         self._set_sub_prop('destinationEncryptionConfiguration', api_repr)
 
+    @property
+    def time_partitioning(self):
+        """google.cloud.bigquery.table.TimePartitioning: Specifies time-based
```

Comment:
I really wish these would render as links, especially with non-built-in types. I know we are waiting on https://github.com/GoogleCloudPlatform/google-cloud-python/issues/5027 , but maybe until that is resolved we should make these into Sphinx references? @jonparrott, thoughts on that?